backend-ci/app/Services/Inventory/StockReconciliationService.php

```
@@ -0,0 +1,144 @@
+                'approved_at' => date('Y-m-d H:i:s'),
+            ]);
+            $db->transCommit();
+        } catch (\Throwable $e) {
+            $db->transRollback();
+            throw $e;
+        }
+
+        return ['success' => true, 'data' => $this->require($id)];
+    }
+
+    public function reject(int $id, array $input): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$input'. (undefined)

(UnusedFormalParameter)